src/fonts.css

```diff
@@ -22,7 +16,7 @@ $text-black: #000000;
 
 html {
   line-height: 1.15;
-  font-size: 14pt;
+  font-size: 32pt;
```

Comment:
why not rem units too?

Reply:
rem units are relative to some root value. Root value must be set to `html`. All other sizes are relative to this one.